feat: Next.js documentation (#2585)
getsentry/sentry-docs: src/includes/getting-started-config/javascript.nextjs.mdx

> If you're using [Vercel](https://vercel.com) you just need to install [Sentry's Vercel integration](/product/integrations/vercel/) and we'll automatically configure your environment for you.

Reviewer: minor smoothing
```suggestion
If you're using [Vercel](https://vercel.com) you only need to install [Sentry's Vercel integration](/product/integrations/vercel/), then we'll automatically configure your environment for you.
```
